elevenlabs_mcp/server.py

```
@@ -0,0 +1,900 @@
+    )
+
+
+@mcp.tool(
+    description="""Clone a voice using provided audio files.
+
+    ⚠️ COST WARNING: This tool makes an API call to ElevenLabs which may incur costs. Only use when explicitly requested by the user.
+    """
+)
+@_safe_api
+def voice_clone(
+    name: str, files: list[str], description: str = None
```

Ruff (0.15.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)